integration/spark/README.md

```diff
@@ -0,0 +1,147 @@
+# OpenLineage Spark Agent
+
+The OpenLineage Spark Agent uses jvm instrumentation to emit OpenLineage metadata.
+
+## Installation
+
+Maven:
+
+```xml
+<dependency>
+    <groupId>io.openlineage</groupId>
+    <artifactId>openlineage-spark</artifactId>
+    <version>0.1.0</version>
+</dependency>
+```
+
+or Gradle:
+
+```groovy
+implementation 'io.openlineage:openlineage-spark:0.1.0'
+```
+
+## Getting started
+
+### Quickstart
+The fastest way to get started testing Spark and OpenLineage is to use the docker-compose files included
+in the project. From the spark integration directory ($OPENLINEAGE_ROOT/integrations/spark) execute
+```bash
+docker-compose up
+```
```

Comment:
This is no longer accurate. There's no local marquez host configured in the docker-compose file- maybe we can just add an instance of marquez in the docker-compose file for the sake of the quickstart?

Reply:
It's good idea as for me, will add it.